src/devices/third_reality.ts

```diff
@@ -186,6 +186,16 @@ const definitions: Definition[] = [
         exposes: [e.battery(), e.temperature(), e.humidity(), e.battery_voltage()],
         ota: ota.zigbeeOTA,
     },
+    {
+        zigbeeModel: ['3RTHS0224BZ'],
+        model: '3RTHS0224BZ',
+        vendor: 'Third Reality',
+        description: 'Temperature and humidity sensor v2',
```

Comment:
Can you remove `toZigbee`, `fromZigbee`, `exposes` and use:

```
extend: [temperature(), battery(), humidity()]
```

Might require the following import:
```
const {temperature, humidity, battery} = require('../lib/modernExtend');
```